projects/core/src/cart/event/multi-cart/multi-cart.event.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Base cart event. Almost all cart events should have `cartId` and `userId`
+ */
+export abstract class MultiCartEvent {
```

Comment:
Why base cart event is named `MultiCartEvent` ?

Reply:
Renamed to CartEvent interface